src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -152,6 +154,7 @@ object FireCloudConfig {
     val trialIndexName = elasticsearch.getString("trialIndex")
     val discoverGroupNames = elasticsearch.getStringList("discoverGroupNames")
     val shareLogIndexName: String = elasticsearch.getString("shareLogIndex")
+    val maxAggregations: Int = Try(elasticsearch.getInt("maxAggregations")).getOrElse(10000)
```

Comment:
your call on preference - instead of a `Try ... getOrElse` you could specify `maxAutocomplete = 1000` in reference.conf and leave this at just `elasticsearch.getInt("maxAggregations")`

Reply:
someone could still set it to an invalid value (i.e. not an int) in the generated conf. unlikely, but still...